bundles/org.openhab.binding.heliosventilation/src/main/resources/ESH-INF/i18n/heliosventilation_de.properties

```
@@ -0,0 +1,79 @@
+# binding
+binding.heliosventilation.name = Helios KWL Binding
+binding.heliosventilation.description = Dies ist das Binding f�r Helios KWL Systeme KWL EC 200/300/500 Pro. Es ben�tigt eine Verbindung zum RS485 Bus f�r die Fernbedienung KWL-FB (9417).
+
+# thing types
+thing-type.heliosventilation.ventilation.label = KWL
+thing-type.heliosventilation.ventilation.description = L�ftungsger�t zur kontrollierten Wohnrauml�ftung.
+
+# thing type config description
+thing-type.config.heliosventilation.ventilation.serialPort.label = Serielle Schnittstelle f�r RS485-Bus
```

Label always start with upper-case
```suggestion
thing-type.config.heliosventilation.ventilation.serialPort.label = Serielle Schnittstelle
```

---

I don't fully understand your comment. Cause I didn't find any lower case label, the suggested change is good for sure, thanks!